SwiftReflector/SwiftXmlReflection/ProtocolDeclaration.cs

```diff
@@ -48,6 +49,78 @@ public AssociatedTypeDeclaration AssociatedTypeNamed (string name)
 		{
 			return AssociatedTypes.FirstOrDefault (at => at.Name == name);
 		}
+
+		public bool HasDynamicSelf {
+			// you could cache this, but this type is not mutable, so that would be bad
```

Comment:
Is this comment correct? Sounds like caching something would be bad if the type were mutable, not if it weren't.

Reply:
Good catch.